connector/src/main/java/org/jboss/as/connector/subsystems/common/pool/Constants.java

```diff
@@ -98,54 +99,62 @@
             .setMeasurementUnit(MeasurementUnit.MINUTES)
             .setAllowExpression(true)
             .setValidator(new IntRangeValidator(0, Integer.MAX_VALUE, true, true))
+            .setRestartAllServices()
             .build();
 
     public static final SimpleAttributeDefinition BACKGROUNDVALIDATIONMILLIS = new SimpleAttributeDefinitionBuilder(BACKGROUNDVALIDATIONMILLIS_NAME, ModelType.LONG, true)
             .setXmlName(Validation.Tag.BACKGROUND_VALIDATION_MILLIS.getLocalName())
             .setMeasurementUnit(MeasurementUnit.MILLISECONDS)
             .setValidator(new IntRangeValidator(1, true, true))
             .setAllowExpression(true)
+            .setRestartAllServices()
             .build();
 
     public static final SimpleAttributeDefinition BACKGROUNDVALIDATION = new SimpleAttributeDefinitionBuilder(BACKGROUNDVALIDATION_NAME, ModelType.BOOLEAN, true)
             .setXmlName(Validation.Tag.BACKGROUND_VALIDATION.getLocalName())
             .setAllowExpression(true)
             //.setDefaultValue(new ModelNode(Defaults.BACKGROUND_VALIDATION))
+            .setRestartAllServices()
             .build();
 
     public static final SimpleAttributeDefinition USE_FAST_FAIL = new SimpleAttributeDefinitionBuilder(USE_FAST_FAIL_NAME, ModelType.BOOLEAN, true)
             .setXmlName(Validation.Tag.USE_FAST_FAIL.getLocalName())
             .setDefaultValue(new ModelNode(Defaults.USE_FAST_FAIL))
             .setAllowExpression(true)
+            .setRestartAllServices()
             .build();
 
     public static SimpleAttributeDefinition VALIDATE_ON_MATCH = new SimpleAttributeDefinitionBuilder(VALIDATEONMATCH_NAME, ModelType.BOOLEAN, true)
-                .setXmlName(Validation.Tag.VALIDATE_ON_MATCH.getLocalName())
-                .setAllowExpression(true)
-                .build();
+            .setXmlName(Validation.Tag.VALIDATE_ON_MATCH.getLocalName())
+            .setAllowExpression(true)
+            .setRestartAllServices()
+            .build();
 
     public static final SimpleAttributeDefinition MAX_POOL_SIZE = new SimpleAttributeDefinitionBuilder(MAX_POOL_SIZE_NAME, ModelType.INT, true)
             .setXmlName(Pool.Tag.MAX_POOL_SIZE.getLocalName())
             .setAllowExpression(true)
             .setDefaultValue(new ModelNode(Defaults.MAX_POOL_SIZE))
+            .setRestartAllServices()
```

Comment:
are you sure this requires reload? There are attributes defined in PoolConfigurationRWHandler that don't require reload.